content/tracing/setup/python.md

```diff
@@ -33,94 +33,59 @@ Next, install the Datadog Tracing library using pip:
 pip install ddtrace
 ```
 
-Finally, import the tracer and instrument your code!
+Then to instrument your Python application use the included `ddtrace-run`
+command. To use it, prefix your Python entry-point command with
+`ddtrace-run`.
 
-## Example
+For example, if your application is started with `python app.py` then:
 
-```python
-
-from ddtrace import tracer
-
-with tracer.trace("web.request", service="my_service") as span:
-  span.set_tag("my_tag", "my_value")
+```sh
+$ ddtrace-run python app.py
 ```
 
-For more examples, see the [Getting Started section of library documentation][2].
+For more advanced usage check out our [API documentation](http://pypi.datadoghq.com/trace/docs/).
```

Comment:
I guess we can write something like: "For more advanced usage and fine-grained control [...]".